php_xdebug.h

```diff
@@ -284,6 +287,11 @@ ZEND_BEGIN_MODULE_GLOBALS(xdebug)
 	zend_bool  do_scream;
 	zend_bool  in_at;
 
+	/* garbage stats */
+	zval gc_runs;
+	int gc_stats_enabled;
+	int gc_show_report;
```

Comment:
these ints should probably be zend_bools  - like other INI settings, that are a bool?